ec2-cluster/modules/cluster-provisioner/main.tf

```diff
@@ -26,6 +26,7 @@ resource "null_resource" "cluster_provisioner" {
 
   provisioner "remote-exec" {
     inline = [
+      "sudo apt-get install jq -y",
```

Comment:
If you modify it, please add `htop` and `lnav`, as both are heavily used too

Reply:
👍🏼